### PR TITLE
⚡ ❓  Make AMP optional

### PR DESCRIPTION
**Pull request by AileenCGN**
refs TryGhost/Ghost#7769

Adds a checkbox to disable AMP support in general settings.

### CODE REVIEW

**Comment by AileenCGN** [comment]
@kevinansfield Can't assign you, but could you please review that?

**Comment by kevinansfield** [comment]
👍 sure

File: app/mirage/fixtures/settings.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ export default [
     {
         created_at: '2015-09-11T09:44:30.805Z',
         created_by: 1,
-        id: 1,
+        id: 7,
```
Comment (kirrg001) [inline app/mirage/fixtures/settings.js]: PR works great. Just out of curiosity: why all the id changes? 🕵🏻

Comment (AileenCGN) [inline app/mirage/fixtures/settings.js]: Just me being 🐷... Didn't know which ID to use for `amp`, so I updated them regarding the ID from our API response.

**Pull request merged**